### PR TITLE
Updated issue template

### PR DESCRIPTION
**Pull request by CandyFace**
This should hopefully make it easier to understand where it's possible to write.
The result of this will also be that any new issues that does not follow the template can and will likely be closed on the spot unless the author edits the description accordingly to the template.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -1,36 +1,64 @@
-<!--
-Use Markdown language for styling.
-Github's Markdown Guide https://guides.github.com/features/mastering-markdown/
-Please use the SEARCH function before opening a new issue!
+<!-- 
+Make sure that the issue you're about to make does not exist already, use the search function to find possible duplicates.
+
+*OBS: Reports that does not follow the template below, will be considered invalid and closed. :OBS*
```
Comment (scribblemaniac) [inline .github/ISSUE_TEMPLATE.md]: What is OBS supposed to be? It is not clear.

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -1,36 +1,64 @@
...
+<!-- WRITE ABOVE THIS LINE -----
+hint: describe the exact steps to reproduce the problem as best as you can
+------ OR BELOW THIS --> 
 
 ### --System Information--
 + Pencil2D Version:
-<!-- You can find the VERSION in Menu->Help->About, e.g., 0.5.4 or Nightly build 02 June 2017) -->
+
+<!-- WRITE ABOVE THIS LINE -----
+hint: You can find the VERSION in Menu->Help->About, e.g., 0.5.4 or Nightly build 02 June 2017)
```
Comment (scribblemaniac) [inline .github/ISSUE_TEMPLATE.md]: Unmatched parenthesis at the end here

File: .github/ISSUE_TEMPLATE.md [hunk 3]
```diff
@@ -1,36 +1,64 @@
...
+------ OR BELOW THIS --> 
 
 + Operating System:
-<!-- (e.g., Windows10, macOS 10.12, Ubuntu 16.04) -->
+
+<!-- WRITE ABOVE THIS LINE -----
+hint: Windows10, macOS 10.12, Ubuntu 16.04
```
Comment (scribblemaniac) [inline .github/ISSUE_TEMPLATE.md]: Can we add etc. to the end of this?

Comment (scribblemaniac) [reply .github/ISSUE_TEMPLATE.md]: Windows10 should probably have a space

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (CandyFace) [reply .github/ISSUE_TEMPLATE.md]: I was referring to Observe but I think I'll just remove it.

**Pull request merged**